.github/workflows/codacy-coverage-reporter.yaml

```
@@ -0,0 +1,16 @@
+jobs:
+  codacy-coverage-reporter:
+    runs-on: ubuntu-latest
+    name: codacy-coverage-reporter
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run codacy-coverage-reporter
+        uses: codacy/codacy-coverage-reporter-action@v1.3.0
+        with:
+          api-token: ${{ secrets.CODACY_API_TOKEN }}
+          coverage-reports: coverage.xml
+    
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)